From ae2bbf1b647b62ad298fefe6cff785683b0d3c87 Mon Sep 17 00:00:00 2001
From: Sivaprasad Tummala <sivaprasad.tummala@amd.com>
Date: Fri, 31 Oct 2025 04:42:28 +0000
Subject: [PATCH 1/2] cmake: fix mismatched package name in
 find_package_handle_*

Fix CMake warning about inconsistent package naming between
`find_package` and `find_package_handle_standard_args`.
The previous code used `Vpp` instead of `VPP`, triggering:

The package name passed to `find_package_handle_standard_args`
(Vpp) does not match the name of the calling package (VPP).

Using consistent capitalization ensures correct variable naming
(e.g., VPP_FOUND) and avoids developer warnings during configuration.

Signed-off-by: Sivaprasad Tummala <sivaprasad.tummala@amd.com>
---
 cmake/FindVPP.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/FindVPP.cmake b/cmake/FindVPP.cmake
index 7d8d16d..a65e163 100644
--- a/cmake/FindVPP.cmake
+++ b/cmake/FindVPP.cmake
@@ -71,4 +71,4 @@ set(VPP_LIBRARIES ${VPP_LIBRARY_MEMORYCLIENT} ${VPP_LIBRARY_SVM} ${VPP_LIBRARY_I
 set(VPP_INCLUDE_DIRS ${VPP_INCLUDE_DIR} ${VPP_INCLUDE_DIR}/vpp_plugins)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(Vpp DEFAULT_MSG VPP_LIBRARIES VPP_INCLUDE_DIRS)
+find_package_handle_standard_args(VPP DEFAULT_MSG VPP_LIBRARIES VPP_INCLUDE_DIRS)
-- 
2.34.1

